core/src/main/java/org/mskcc/oncotree/utils/TumorTypesUtil.java

```diff
@@ -137,33 +136,60 @@ public static InputStream getTumorTypeInputStreamFromGitHub(Version version) {
         return tumorTypes;
     }
 
-    private static Map<String, TumorType> parseFromRaw(InputStream inputStream, Version version) {
-        TsvParserSettings settings = new TsvParserSettings();
+    private static Map<String, TumorType> loadFromRepository(Version version) {
+        List<OncoTreeNode> oncoTreeNodes = oncoTreeRepository.getOncoTree();
+        Map<String, TumorType> tree = new HashMap<>();
 
-        //the line separator sequence is defined here to ensure systems such as MacOS and Windows
-        //are able to process this file correctly (MacOS uses '\r'; and Windows uses '\r\n').
-        settings.getFormat().setLineSeparator("\n");
-
-        // creates a TSV parser
-        TsvParser parser = new TsvParser(settings);
-
-        Map<String, TumorType> tumorTypes = new HashMap<>();
+        TumorType root = new TumorType();
+        root.setCode("TISSUE");
```

Comment:
The spreadsheet had no tissue node, but if TISSUE is a valid oncotree code (maybe we need to confirm this), then it could/should have meta information (color, mainType) which can be set inside of topbraid and which we should try to extract and use.